master/security/kubernetes-default-deny.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Enable default deny for Kubernetes pods
+---
+
+### Big picture
+
+Change the default for Kubernetes pods from allowing all traffic, to denying all traffic using Kubernetes or Calico network policy.  
+
+### Value
+
+A **default deny** network policy provides an enhanced security posture -- so workloads without policy (or incorrect policy) do not allow traffic until appropriate network policy is defined.
```

Comment:
nit "do not allow" -> "are not allowed"

The workloads themselves aren't in control of which traffic they receive / send.

Reply:
Fixed. good one.